xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm

```diff
@@ -49,10 +49,9 @@ $xwiki.ssfx.use('uicomponents/pagination/pagination.css', true)##
 ##
 ##
     #set ($formname = 'historyform')
-    <form id="$formname" action="$doc.getURL('view', "viewer=changes&amp;$docvariant")" method="post">
+    <form id="$formname" action="$doc.getURL('view', "viewer=changes&amp;$docvariant")" method="get">
       <div id="_history">
       ## CSRF prevention
-      <input type="hidden" name="form_token" value="$!{services.csrf.getToken()}" />
```

Comment:
Why did you remove the CSRF token passing? I think the CSRF protection will fail as a consequence.

Reply:
Forget that, just read the jira issue and saw the comment from @evalica

Reply:
@gdelhumeau : do you remember why you put a CSRF token and why you used POST? https://github.com/xwiki/xwiki-platform/commit/98179813d56aeb9eb07f55aea69436e61c6ddc54#diff-a894b9fa96163e4b3286f4e13ef18463R52